searx/utils.py

```diff
@@ -538,7 +531,7 @@ def get_xpath(xpath_spec):
     raise TypeError('xpath_spec must be either a str or a lxml.etree.XPath')
 
 
-def eval_xpath(element, xpath_spec):
+def eval_xpath(element: ElementBase, xpath_spec: str) -> Any:
```

Comment:
I think in general no type annotation is better than `Any`.

Reply:
Removed